examples/TrafficLightNeoPixel/TrafficLightNeoPixel.ino

```diff
@@ -0,0 +1,56 @@
+#include <Arduino.h>
+#include <MycilaTrafficLight.h>
+
+// Which pin is connected to the NeoPixels?
+constexpr uint16_t NEOPIXEL_PIN = 4;
+
+// How many NeoPixels 
+constexpr uint16_t NEOPIXEL_COUNT = 3 ;
+
+// Pixel type flags, add together as needed:
+//   NEO_KHZ800  800 KHz bitstream (most NeoPixel products w/WS2812 LEDs)
+//   NEO_KHZ400  400 KHz (classic 'v1' (not v2) FLORA pixels, WS2811 drivers)
+//   NEO_GRB     Pixels are wired for GRB bitstream (most NeoPixel products)
+//   NEO_RGB     Pixels are wired for RGB bitstream (v1 FLORA pixels, not v2)
+//   NEO_RGBW    Pixels are wired for RGBW bitstream (NeoPixel RGBW products)
+constexpr neoPixelType NEOPIXEL_TYPE = NEO_GRB + NEO_KHZ800 ;
+
+Mycila::TrafficLight lights;
+
+void setup() {
+  Serial.begin(115200);
+  while (!Serial)
+    continue;
+
+  // Argument 1 = pin number (most are valid)
+  // Argument 2 = Number of pixels in NeoPixel strip (default 3 if omited)
+  // Argument 3 = Pixel type flags defaut if omited ( NEO_GRB + NEO_KHZ800 )
+  lights.begin( NEOPIXEL_PIN, NEOPIXEL_COUNT, NEOPIXEL_TYPE ); 
+
+  // Set BRIGHTNESS to about 1/5 (max = 255)
+  lights.setBrightness(50); 
```

Comment:
here I would prefer to have a float `[0.0, 1.0]`, because it will cover any use case (even future ones).
the limit of 255 is implementation specific for now (even if we could argue that it matches a 8 bit brightness) so the level could then be found by multiplying the % with the limit.